src/controllers/project.controller.ts

```diff
@@ -119,13 +121,68 @@ export class ProjectController {
 
 		this.view.update({ type: MessageType.setSessionsSection, payload: sessionsViewObject });
 
+		const activeDeployment = deployments?.find((deployment) => deployment.state === DeploymentState.ACTIVE_DEPLOYMENT);
+
 		if (this.selectedDeploymentId) {
 			await this.selectDeployment(this.selectedDeploymentId);
 		}
+
+		if (!activeDeployment) {
```

Comment:
Check if we want to shoot the session - only to the activeDeployment.

Reply:
We can't get the code of the previous versions. We should singleshot only the last deployment.